Bug: `test_everflow_ipv6`'s first testlets can fail due to `AssertionError: Did not receive expected packet`

### Is it platform specific

generic

### Importance or Severity

High

### Description of the bug

When running `everflow/test_everflow_ipv6.py` we sometimes see the first testlest fail with `AssertionError: Did not receive expected packet`

This is because the `setup_info` fixture shuts down bgp via `sudo config bgp shutdown all` but doesn't wait for it to complete, it just sleeps for 60s hoping it's enough time:
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/everflow/everflow_test_utilities.py#L434-L445
```
    # Disable BGP so that we don't keep on bouncing back mirror packets
    # If we send TTL=1 packet we don't need this but in multi-asic TTL > 1

    if 't2' in topo and 'lt2' not in topo and 'ft2' not in topo:
        for dut_host in duthosts.frontend_nodes:
            dut_host.command("sudo config bgp shutdown all")
            dut_host.command("mkdir -p {}".format(DUT_RUN_DIR))
    else:
        duthost.command("sudo config bgp shutdown all")
        duthost.command("mkdir -p {}".format(DUT_RUN_DIR))

    time.sleep(60)
```

I tried this out on a 64-port Q3D device and I can see the OA is busy for ~2.5minutes when shutting down bgp:
```
> sudo config bgp shutdown all

2026 Mar 11 22:53:12.929273 ctn101 DEBUG bgp#bgpcfgd: Received message : '('10.0.0.1', 'SET', (('admin_status', 'down')
...
2026 Mar 11 22:55:25.960720 ctn101 NOTICE swss#orchagent: :- removeNextHopGroup: Delete next hop group
```

If we move onto the next test steps (setting up mirror session and ACL) while routes are still being removed we can see that the Mirror session comes up and ACLs come up momentarily against the routes that are disappearing:
```
2026 Mar 11 19:15:53.863493 ctn101 INFO python3.13[179622]: ansible-ansible.legacy.command Invoked with _raw_params=config mirror_session add test_session_1 1.1.1.1 2.2.2.2 8 4 35006
2026 Mar 11 19:16:01.648351 ctn101 INFO python3.13[180533]: ansible-ansible.legacy.command Invoked with _raw_params=acl-loader update full /tmp/everflow/acl-erspan.json --table_name EVERFLOWV6 --session_name test_session_1

...

2026 Mar 11 19:16:01.935205 ctn101 NOTICE swss#orchagent: :- updateNextHop: Updating mirror session test_session_1 with route 0.0.0.0/0
2026 Mar 11 19:16:01.935205 ctn101 NOTICE swss#orchagent: :- updateNextHop:     next hop IPs: 10.0.0.7@PortChannel108,10.0.0.9@PortChannel110,10.0.0.57@PortChannel158,10.0.0.59@PortChannel160,10.0.0.61@PortChannel162,10.0.0.63@PortChannel164
2026 Mar 11 19:16:01.935205 ctn101 NOTICE swss#orchagent: :- updateNextHop: Updated mirror session state db test_session_1 nexthop to 10.0.0.7@PortChannel108
2026 Mar 11 19:16:01.935238 ctn101 NOTICE swss#orchagent: :- getNeighborInfo: Mirror session test_session_1 neighbor is PortChannel108

...

2026 Mar 11 19:16:02.557394 ctn101 NOTICE swss#orchagent: :- add: Successfully created ACL rule RULE_1 in table EVERFLOWV6
…
2026 Mar 11 19:16:03.127981 ctn101 NOTICE swss#orchagent: :- add: Successfully created ACL rule RULE_26 in table EVERFLOWV6
```

The mirror session coming up and ACLs being programmed trick the test into moving to the next step of injecting the packets, but because routes are still being removed eventually the mirror session and ACLs will go down:
```
2026 Mar 11 19:16:08.799680 ctn101 NOTICE swss#orchagent: :- removeNextHopGroup: Delete next hop group 10.0.0.7@PortChannel108,10.0.0.9@PortChannel110,10.0.0.53@PortChannel154,10.0.0.55@PortChannel156,10.0.0.57@PortChannel158,10.0.0.59@PortChannel160,10.0.0.61@PortChannel162,10.0.0.63@PortChannel164
2026 Mar 11 19:16:09.272831 ctn101 NOTICE swss#orchagent: :- updateNextHop: Updating mirror session test_session_1 with route 0.0.0.0/0
2026 Mar 11 19:16:09.272831 ctn101 NOTICE swss#orchagent: :- updateNextHop: Updated mirror session state db test_session_1 nexthop to 0.0.0.0@
2026 Mar 11 19:16:09.681114 ctn101 NOTICE swss#orchagent: :- deactivateSession: Deactivated mirror session test_session_1
```

After route churn completes and our test installed route is installed successfully the mirror session and ACLs will come back up:
```
2026 Mar 11 19:15:54.718796 ctn101 INFO python3.13[179770]: ansible-ansible.legacy.command Invoked with _raw_params=vtysh -c "configure terminal" -c "ip route 2.2.2.0/24 10.0.0.143 tag 1"

...

2026 Mar 11 19:17:02.650085 ctn101 NOTICE swss#orchagent: :- updateNextHop: Updating mirror session test_session_1 with route 2.2.2.0/24
2026 Mar 11 19:17:02.650085 ctn101 NOTICE swss#orchagent: :- updateNextHop:     next hop IPs: 10.0.0.143@Ethernet156
2026 Mar 11 19:17:02.650112 ctn101 NOTICE swss#orchagent: :- updateNextHop: Updated mirror session state db test_session_1 nexthop to 10.0.0.143@Ethernet156
2026 Mar 11 19:17:02.650112 ctn101 NOTICE swss#orchagent: :- getNeighborInfo: Mirror session test_session_1 neighbor is Ethernet156
```

But any test that is injecting packets during the window where the mirror session went back down will fail because no packet is mirrored.

### Steps to Reproduce

Run `everflow/test_everflow_ipv6.py` on a topology with lots of ports (64) and lots of routes (32k per upstream neighbor)

### Actual Behavior and Expected Behavior

The correct solution here would be to wait for the route removal to complete after `sudo config bgp shutdown all` before proceeding with the rest of the test/fixtures.

Thankfully we already have a fixture which handles this in `shutdown_ebgp`:
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/fixtures/duthost_utils.py#L257-L302
I think we should probably just switch to using this fixture.

When using this fixture I can see the appropriate amount of time is being waited before proceeding with the rest of the test(~2.5min):
```
2026 Mar 13 16:53:44.801950 ctn101 INFO python3.13[1125272]: ansible-ansible.legacy.command Invoked with _raw_params=sudo config bgp shutdown all
...
2026 Mar 13 16:53:50.480736 ctn101 INFO python3.13[1126346]: ansible-ansible.legacy.command Invoked with _raw_params=show ip route sum
...
2026 Mar 13 16:54:40.715999 ctn101 INFO python3.13[1132508]: ansible-ansible.legacy.command Invoked with _raw_params=show ip route sum
2026 Mar 13 16:54:42.284513 ctn101 INFO python3.13[1132848]: ansible-ansible.legacy.command Invoked with _raw_params=COLUMNS=512 show processes cpu | grep orchagent | awk '{print $9}'
...
2026 Mar 13 16:56:14.748837 ctn101 INFO python3.13[1144399]: ansible-ansible.legacy.command Invoked with _raw_params=COLUMNS=512 show processes cpu | grep orchagent | awk '{print $9}'
```

### Relevant log output

```shell

```

### Output of `show version`

```shell

```

### Attach files (if any)

_No response_

--- a/tests/everflow/everflow_test_utilities.py
+++ b/tests/everflow/everflow_test_utilities.py
@@ -26,7 +26,7 @@ from tests.common.helpers.constants import UPSTREAM_NEIGHBOR_MAP, DOWNSTREAM_NEI
 from tests.common.macsec.macsec_helper import MACSEC_INFO
 from tests.common.dualtor.dual_tor_common import mux_config              # noqa: F401
 from tests.common.helpers.sonic_db import AsicDbCli
-from tests.common.fixtures.duthost_utils import shutdown_ebgp # noqa: F401
+from tests.common.fixtures.duthost_utils import shutdown_ebgp          # noqa: F401
 import json
 
 logger = logging.getLogger(__name__)
@@ -407,7 +407,8 @@ def get_queue_counters(dut, asic_ns, port, queue):
 
 
 @pytest.fixture(scope="module")
-def setup_info(duthosts, rand_one_dut_hostname, tbinfo, request, topo_scenario, shutdown_ebgp):
+def setup_info(duthosts, rand_one_dut_hostname, tbinfo, request, topo_scenario,
+               shutdown_ebgp):  # noqa: F811
     """
     Gather all required test information.
 

